tests/python/quantization/test_quantization.py

```diff
@@ -283,16 +283,16 @@ def check_quantized_fc(data_shape, num_hidden, no_bias, qdtype, flatten=True):
         fc_fp32_exe = fc_fp32.simple_bind(ctx=mx.current_context(), grad_req='null')
         if qdtype == 'uint8':
             data_low = 0.0
-            data_high = 127.0
+            data_high = 63.0
```

Comment:
Any reason of changing this?

Reply:
Change data range from (-127,127) to (-63, 63) to avoid overflow when using igemm sometimes